tests/testthat/test-add_gloabl_p.tbl_uvregression.R

```diff
@@ -0,0 +1,156 @@
+test_that("add_global_p.tbl_uvregression(x)", {
+  tbl <- trial |>
+    tbl_uvregression(method = lm, y = marker, include = c("age", "grade"))
+
+  expect_silent(
+    res <- tbl |> add_global_p()
+  )
+
+  expect_snapshot(res |> as.data.frame())
+
+  # two model terms, two p values
+  expect_equal(
+    sum(!is.na(res$table_body$p.value)),
+    2
+  )
+
+  # test p-values are properly being calculated
+  expect_equal(
+    res$table_body$p.value[1:2],
+    c(
+      (car::Anova(lm(marker ~ age, trial)))$`Pr(>F)`[1],
+      (car::Anova(lm(marker ~ grade, trial)))$`Pr(>F)`[1]
+    )
+  )
+})
+
+test_that("add_global_p.tbl_uvregression(include)", {
+  tbl <- trial |>
+    tbl_uvregression(method = lm, y = marker, include = c("age", "grade"))
+
+  res1 <- tbl |> add_global_p(include = age)
+
+  # one expected p-value for age
+  expect_equal(
+    sum(!is.na(res1$table_body$p.value)),
+    1
+  )
+})
+
+test_that("add_global_p.tbl_uvregression(type)", {
+  tbl <- trial |>
+    tbl_uvregression(method = lm, y = marker, include = c("age", "grade"))
+
+  res2 <- tbl |> add_global_p(type = "II")
+
+  # 4 expected p-values, 2 for each variable, 2 for each level of grade
+  expect_equal(
```

Comment:
Can we also test that the p-values match the results from `car::Anova(type = "II")`